tango_simlib/tango_sim_generator.py

```diff
@@ -392,6 +393,9 @@ def get_parser_instance(sim_datafile):
     elif extension in [".xml"]:
         parser_instance = SDDParser()
         parser_instance.parse(sim_datafile)
+    elif extension in [".fgo"]:
+        parser_instance = FandangoExportDeviceParser()
+        parser_instance.parse(sim_datafile)
     return parser_instance
 
 def configure_device_model(sim_data_file=None, test_device_name=None):
```

Comment:
Update the docstring to be more generic instead referring to just the """...xmi file.."""